src/app/(proper_react)/redesign/(public)/LandingView.test.tsx

```diff
@@ -388,4 +388,55 @@ describe("When Premium is available", () => {
     const progressCardIllustration = screen.getByTestId("progress-card-image");
     expect(progressCardIllustration).toBeInTheDocument();
   });
+
+  it("opens and closes an FAQ accordion item", async () => {
+    const user = userEvent.setup();
+    const ComposedDashboard = composeStory(LandingUs, Meta);
+    render(<ComposedDashboard />);
+    const faqQuestion = screen.getByText(
```

Comment:
And this would be a `getByRole("button"` with the other changes.

Reply:
Ah, yeah, if you want to do a partial match on text so you can avoid the SVG, you can pass in a regular expression instead of a string. It doesn't need to be a complex regexp; you can still just match on exact characters, e.g. like this:

```ts
    const faqQuestion = screen.getByRole("button",
      { name: /What kinds of websites sell my personal information?/ },
    );
```

And yes, definitely a good idea to verify that `aria-hidden` is set correctly!